content/docs/_index.md

```diff
@@ -4,28 +4,38 @@ weight=1
 summary="Get started with Cloud Native Buildpacks."
 +++
 
-### Tutorials
+## Talks
+
+{{< talks >}}
```

Comment:
LGTM, my only question is whether we want talks to be all the way at the top. I think they might be a better fit after the tools section. I don't have strong opinions either way, but I would prefer some textual documentation that provides some "getting started" steps \w buildpacks at the top before referring to talks.

Reply:
It's a good question. I don't have a strong preference either way. I think the video at the top is best for first-time visitors, but for repeat visitors who are looking for some specific documentation it might not be the right place to start. unsure.

Reply:
@jromero any thoughts on this?

Reply:
I'm definitely trying to tailor to the first time user. I think most second+ visitors would use the left nav bar to jump to where they'd want.